docs/en/modules/net.md

```diff
@@ -282,7 +338,7 @@ end)
 
 ## net.socket:unhold()
 
-Unblock TCP receiving data, i.e. undo `hold()`.
+Unblock TCP receiving data. I.e. undo `hold()`.
```

Comment:
Probably, but the original punctuation messes up [the table at RTD](https://nodemcu.readthedocs.org/en/dev/en/modules/net/#net-module). If you have a better proposal - I didn't spend too much thoughts on the fix.